src/data/models/character.ts

```diff
@@ -0,0 +1,40 @@
+import mongoose from 'mongoose'
```

Comment:
I think that you should be careful with models name.
You have for example the User model for mongo but also a User model for TS.
You have to distinguish them when naming the files at least.
For example you could name this file character-shcema.ts and the TS model just character.ts

Reply:
Will do!